### PR TITLE
Substitute `hr` for `div` (#81 fix)

### LINKED ISSUE #81: Tiniest and most annoying issue ever
right border edge of the progress item box

image

### PR DESCRIPTION
**Pull request by carlos-menezes**
Fixes #81 

fix

### CODE REVIEW

File: components/style.css [hunk 1]
```diff
@@ -163,7 +163,7 @@ hr {
 }
 .progress-item-separator {
   margin: 0px;
-  width: 100%;
```
Comment (Southclaws) [inline components/style.css:166]: this feels hacky, but I'm not sure what the actual solution would be.

**Comment by carlos-menezes** [comment]
What we can do is sub out the `hr` for a `div`.

**Comment by Southclaws** [comment]
That might make more sense tbh.

**Comment by carlos-menezes** [comment]
Fixing an error through your phone is painful.

**Comment by Southclaws** [comment]
Well it passed CI tests so congrats!

**Comment by carlos-menezes** [comment]
😂

Cheers.

**Pull request merged**